<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPlayerDialog</class>
 <widget class="QDialog" name="addPlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>381</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="hanLE">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="refLE">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Speed</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Passing</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Heading</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="tacLE"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="pasLE"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Agility</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Finishing</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Handling</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="finLE"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="speLE"/>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="outLE"/>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="tecLE"/>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Technique</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="posLE"/>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="strLE"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Tackling</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Out of Area</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Reflexes</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="agiLE"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Positioning</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Strength</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="heaLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>381</width>
     <height>157</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>from excel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="excelLE"/>
      </item>
      <item>
       <widget class="QPushButton" name="parseExcel">
        <property name="text">
         <string>Parse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>or</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLE"/>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="idLE"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ageLE"/>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="roleLE"/>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Caps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="capLE">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>TL date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="tlDate">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="date">
         <date>
          <year>2013</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::DaySection</enum>
        </property>
        <property name="displayFormat">
         <string>d/M/yyyy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Last cap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="capDate">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="date">
         <date>
          <year>2013</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::MonthSection</enum>
        </property>
        <property name="displayFormat">
         <string>d/M/yyyy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Talent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="talentLE"/>
      </item>
      <item>
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Country</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="countryLE"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addPlayerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addPlayerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
